boards/arc/emsdp/emsdp_defconfig

```diff
@@ -17,3 +17,4 @@ CONFIG_ARC_MPU_ENABLE=y
 CONFIG_GPIO=y
 CONFIG_SPI=y
 CONFIG_PINCTRL=y
+CONFIG_I2C=y
```

Comment:
I'm not sure if this is correct.

There were several PRs which eliminates such symbols enablement on the boards level. I.e: https://github.com/zephyrproject-rtos/zephyr/pull/48782

As I unserstand, the idea it that symbols like `CONFIG_I2C`, `CONFIG_SPI`, etc... sholdn't be enabled on the board level but shold be enable by application / sample / test itself if corresponding subsystem is required.

Otherwise we just unnesesary increase image size for any application even if the peripheral isn't ised.

Reply:
Before removing this we need to check if we have at least some test/samle which enables I2C and which is run on emsdp - otherwise we may get this I2C code completely untested (even the device initialization).

If we don't have any test/smaple we probably can modify `tests/drivers/i2c/i2c_tca954x` for our case - and just check that provided I2C devices are ready.

Reply:
We MPU6050 sensor for testing on emsdp board